packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -311,7 +311,11 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                         },
                         "createBlob",
                     );
-                    event.end({ blobId: res.content.id });
+                    event.end({
+                        blobId: res.content.id,
+                        serverRetries: res.headers.get("x-fluid-retries") ?? undefined,
```

Comment:
@vladsud - Not sure that `?? undefined` does what you think - `a ?? b` is different from `a ? a : b`, it doesn't do type coercion on strings/numbers/booleans but is only checking for `null` or `undefined`.  See [this playground example](https://www.typescriptlang.org/play?ts=4.1.3#code/MYewdgzgLgBANiA5jAvDUkRwKYDoGIDcAUMQQBRgCuccMA-PTAETW3MCUJFVYAJtgBmASzDY+DJs14CRYvp25JyzZpJbYAtgAcoAT0VllABnXMAXtgBOIQxUEBDOBGxnHz7IaOJKNOlLY4NQAuFgBHO2UZIVFxBhZouXEQ8MifVXjmLV0DGFDmCK5vclMpSxsUgrTydxdM2s881KKgA).

Reply:
That's precisely what I need - convert null to undefined.